splice_si_api/src/main/java/com/splicemachine/si/impl/server/SITransactor.java

```diff
@@ -284,6 +284,17 @@ private void releaseLocksForKvBatch(Pair<KVPair, Lock>[] locks){
             ConflictResults conflictResults=ConflictResults.NO_CONFLICT;
             KVPair kvPair=baseDataAndLock.getFirst();
             KVPair.Type writeType=kvPair.getType();
+
+            DataResult possibleConflicts = null;
+            if(!skipConflictDetection)
+            {
+                // we need the possibleConflicts for
+                // - has constraintChecker && writeType == UPSERT/UPDATE/DELETE: checking for constraint violations
+                // - writeType == INSERT/UPSERT/DELETE: evaluating addFirstOccurrenceToken
+                if( bloomInMemoryCheck == null || bloomInMemoryCheck.get(i) ) {
+                    possibleConflicts = table.getLatest(kvPair.getRowKey(), possibleConflicts);
```

Comment:
note for myself: are these conflicts collected? why does getLatest have parameter possibleConfilcts?